dw-snackbar.js

```diff
@@ -328,17 +336,29 @@ export class DwSnackbar extends layoutMixin(LitElement) {
     }
     return html`<dw-button
       .label="${config.actionButton.caption}"
-      ?disabled=${this._disableButton}
+      ?disabled=${config.actionButtondisabled}
       @click="${() => {
         this._onAction(config);
       }}"
     ></dw-button>`;
   }
 
   async _onAction(config) {
-    this._disableButton = true;
+    this._toastList = {
```

Comment:
We shouldn't update it's configuration either. Instead, simple fix was that there should be registry which keeps record of the toasts for whom action buttons are disabled. This registry can be a property, but it's data type should be `Object` instead you took `Boolean` earlier. This `Object` is really a Hash, key = toastId, value = boolean - whether buttons are disabled or not.

This property should be updated in immutable way; and everything can work properly. Please, apply this fix.